Replaces Security Voidsuit helmet sprites.

juxjux9930 opened the pull request:
🆑 Kozak411
rscadd: Resprites the Torch's Security voidsuit helmets
/🆑

The Torch's special voidsuits generally have a theme of larger visors on a round-ish helmet with the department colors somewhere on the side or top. Security was the only outlier with their deadspace-esque helmet. New sprites change that.

juxjux9930 committed:
Replaces Security Voidsuit helmet sprites.

The Sec-Alt voidsuit that the Torch uses has a helmet that doesn't match the rest of the Torch suits and just looks kind of doofy. These new sprites are more in-theme.

Spookerton commented:
Erroneously deletes salvage helmets. Not convinced this *actually looks good* either, at the moment.

juxtaposed9930 commented:
Wait, it deletes salvage helmets? That's.. a problem
Give me a bit to figure that out

juxtaposed9930 commented:
Maybe I'm blind, but I'm not seeing where salvage helmets are deleted. They didn't exist for skrell and unathi and are still there for humans.

juxtaposed9930 commented:
I am blind. Found what you mean, will fix.

juxtaposed9930 commented:
It's fixed

juxjux9930 committed:
Removes the Fire Axe from Bridge Storage

The Torch's bridge has two axes closer together than almost any other place on the ship, with less overall travel time between the two. The placement of the one in bridge storage, right next to two weapons lockers, could make some players believe it is to be used as a defensive weapon. Removing this axe fixes that little problem. And in an emergency where a fireaxe is *actually* necessary, the saferooms should be unbolted and the other fire axe will be available.

juxjux9930 committed:
Merge remote-tracking branch 'origin/dev' into dev

juxjux9930 committed:
Merge remote-tracking branch 'origin/dev' into dev

juxtaposed9930 commented:
Changes sprites a bit. Goggles are so *out*, Y-visor is *in*
